populator-machinery/controller_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"testing"
 	"time"
 
+	// "github.com/google/go-cmp/cmp"
```

Comment:
This line should be removed if it is commented out?

Reply:
Thanks for catching this! Removed